tests/client-sdk/agents/get_boiling_point_tool.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
rename this as `test_custom_tools.py` or something like this?

Reply:
in fact, can we delete this since a test like this is inside `test_agents.py` also in the same directory?

Reply:
done